conf/script/prov-ha-reset

```diff
@@ -54,7 +54,9 @@ resources=(
     ip-c{2,1}
 )
 for r in ${resources[@]}; do
-    pcs resource delete $r || true
+    if [[ $(pcs resource show $r 2> /dev/null) ]]; then
+        pcs resource delete $r || true
```

Comment:
`|| true` in the end makes `if .. then` check redundant since `pcs resource delete` failure will be ignored anyway.
I would suggest to keep this file as is.

Reply:
Yes, we need not add a check, just thought of still having it just in case if the error is parsed by some external script like provisioner. May be then let's just re-direct the error to /dev/null.

Reply:
Actually you are right, let's just keep it as the original.

Reply:
Done.